moveit_ros/move_group/src/move_group_context.cpp

```diff
@@ -40,14 +40,26 @@
 #include <moveit/plan_execution/plan_execution.h>
 #include <moveit/plan_execution/plan_with_sensing.h>
 
+ros::NodeHandle move_group::MoveGroupContext::getConfigNodeHandle()
+{
+  ros::NodeHandle nh = ros::NodeHandle("~");
+  // Handle new configuration scheme with multiple pipelines configured in namespaces planning_pipelines/*
+  std::string default_pipeline;
+  if (nh.getParam("default_planning_pipeline", default_pipeline) &&
+      nh.hasParam("planning_pipelines/" + default_pipeline))
+    nh = ros::NodeHandle(nh, "planning_pipelines/" + default_pipeline);
+  return nh;
+}
+
 move_group::MoveGroupContext::MoveGroupContext(
     const planning_scene_monitor::PlanningSceneMonitorPtr& planning_scene_monitor, bool allow_trajectory_execution,
     bool debug)
   : planning_scene_monitor_(planning_scene_monitor)
   , allow_trajectory_execution_(allow_trajectory_execution)
   , debug_(debug)
 {
-  planning_pipeline_.reset(new planning_pipeline::PlanningPipeline(planning_scene_monitor_->getRobotModel()));
+  planning_pipeline_.reset(
+      new planning_pipeline::PlanningPipeline(planning_scene_monitor_->getRobotModel(), getConfigNodeHandle()));
```

Comment:
This will change the namespace of `display_planned_path` and thus break rviz configs in melodic.

Reply:
Also, should this be switched to `std::make_shared`?

Reply:
Switching to `std::make_shared` is unrelated to this PR topic, isn't it?